### PR TITLE
Changes needed for Fedora 21

### PR DESCRIPTION
**Pull request by tjyang**
Hi
This is just a FYI not a real pull request.

I am using Fedora 21 with go 1.3.3. Looks like gofmt doesn't support "-tabs=false -tabwidth=4 " syntax anymore. Also bootparam.h is in /usr/include/asm/ not /usr/include/x86_64-linux-gnu/asm/.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Perhaps an issue tracker needs to be created.

**Comment by tjyang** [comment]
I can not issue tab on https://github.com/google/novm

**Comment by rgbkrk** [comment]
Sorry, that was directed more at the maintainers since the issue tab isn't available yet you wanted to post an issue.

**Comment by rgbkrk** [comment]
Annnd now it's there.

**Comment by amscanne** [comment]
Thanks @tjyang for the pull request! I'll address these issues shortly.

I've also enabled the issue tracker.

File: Makefile [hunk 1]
```diff
@@ -58,8 +58,8 @@ go-bench:
 	$(call go_build,go test -bench=".*" novmm)
 	$(call go_build,go test -bench=".*" noguest)
 go-fmt:
-	$(call go_build,gofmt -l=true -w=true -tabs=false -tabwidth=4 src/novmm/$*)
-	$(call go_build,gofmt -l=true -w=true -tabs=false -tabwidth=4 src/noguest/$*)
+	$(call go_build,gofmt -l=true -w=true  src/novmm/$*)
+	$(call go_build,gofmt -l=true -w=true  src/noguest/$*)
```
Comment (penberg) [inline Makefile:62]: This looks like totally unrelated to the fix.

File: src/novmm/loader/linux_setup.go [hunk 2]
```diff
@@ -21,9 +21,11 @@ package loader
 #include <string.h>
 
 #ifdef __x86_64__
-#include <x86_64-linux-gnu/asm/bootparam.h>
+//#include <x86_64-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
 #else
-#include <i386-linux-gnu/asm/bootparam.h>
+//#include <i386-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
```
Comment (penberg) [inline src/novmm/loader/linux_setup.go:28]: The #ifdef no longer makes sense and you can just unconditionally `#include asm/bootparam.h`.

File: src/novmm/loader/linux_setup.go [hunk 3]
```diff
@@ -69,75 +71,75 @@ static inline void set_header(
...
-    "novmm/machine"
-    "novmm/platform"
-    "unsafe"
+	"novmm/machine"
+	"novmm/platform"
+	"unsafe"
 )
 
 func SetupLinuxBootParams(
-    model *machine.Model,
```
Comment (penberg) [inline src/novmm/loader/linux_setup.go:78]: Looks like you reformatted the whole file with `gofmt` inserting tabs instead of four spaces for indent.

File: Makefile (hunk 1 above)
Comment (tjyang) [inline Makefile:62]: this Makefile change is for https://github.com/google/novm/issues/3

File: src/novmm/loader/linux_setup.go (hunk 2 above)
Comment (tjyang) [inline src/novmm/loader/linux_setup.go:28]: Thanks, I got it compiled for Fedora, hoping you can come up solution from upstream for both Fedora and Ubuntu.

File: src/novmm/loader/linux_setup.go (hunk 3 above)
Comment (tjyang) [inline src/novmm/loader/linux_setup.go:78]: On Wed, Jan 7, 2015 at 3:02 PM, Pekka Enberg notifications@github.com
wrote:

> In src/novmm/loader/linux_setup.go
> https://github.com/google/novm/pull/1#discussion-diff-22615558:
> 
> >  )
> > 
> >  func SetupLinuxBootParams(
> > -    model *machine.Model,
> 
> Looks like you reformatted the whole file with gofmt inserting tabs
> instead of four spaces for indent.
> 
> Yeah, I can't get "gofmt -l=true -w=true -tabs=false -tabwidth=4
> src/novmm/*"  on Fedora 21 to work. "-tabs" is not valid.

[tjyang@fedora21 novm]$ gofmt -l=true -w=true -tabs=false -tabwidth=4
src/novmm/*
flag provided but not defined: -tabs
usage: gofmt [flags] [path ...]
  -cpuprofile="": write cpu profile to this file
  -d=false: display diffs instead of rewriting files
  -e=false: report all errors (not just the first 10 on different lines)
  -l=false: list files whose formatting differs from gofmt's
  -r="": rewrite rule (e.g., 'a[b:len(a)] -> a[b:]')
  -s=false: simplify code
  -w=false: write result to (source) file instead of stdout
[tjyang@fedora21 novm]$

tj

> —
> Reply to this email directly or view it on GitHub
> https://github.com/google/novm/pull/1/files#r22615558.

## 

T.J. Yang

**Comment by amscanne** [comment]
Fixed via #2 and #3.
